### DISCUSSION TITLE
When will the next version be released? Hopefully with #648 it will solve my problem.

### DISCUSSION THREAD

**Discussion by 0xZhangKe** [Foso/Ktorfit#677] [Q&A] [answered]
When will the next version be released? Hopefully with [#648](https://github.com/Foso/Ktorfit/pull/648) it will solve my problem.

**Comment by Foso** [comment, marked as answer]
That is already released with Ktorfit 2.1.0 https://github.com/Foso/Ktorfit/blob/master/docs/CHANGELOG.md#210
